### DISCUSSION TITLE
Loco Positioning tab on crazyflie client and crazyflie 2.1

### DISCUSSION THREAD

**Discussion by aelvis237** [bitcraze/discussions#1283] [Q&A]
hi,
I've just started working with the crazyflie system so I apologise if my
problem is trivial. I am working with the Loco positioning system i have
set up the Anchors using the steps explained
here:https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/
the problem is the loco positioning tab is blended(i cant see the Nodes or
choose a status.) the firmware on the Crazyflie is updated(2024). i dont
know if its because the lps-firmware(2022.9) is not of 2024 as that of the
crazyflie. Am using Windows 10, i installed the cfclient  using pip :pip3 install cfclient ,and my python version is 3.12
Can anybody help me out here?
Thanks
![cfclient](https://github.com/bitcraze/discussions/assets/62871491/c5a47948-8562-4e44-b0ac-52ad18b26da9)

**Comment by gemenerik** [comment]
I see your address selection is not grayed out, so that suggests you may not have [connected to your Crazyflie](https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/#open-the-crazyflie-client)?

**Reply by aelvis237** [reply]
hi,
thanks for the quick response .Even when am connected it still not change.
![image](https://github.com/bitcraze/discussions/assets/62871491/d0a522e9-6a23-417f-a41c-cd8e7f7c04dd)

**Reply by gemenerik** [reply]
Did you correctly mount your loco positioning deck? Do you see a printout in the console tab saying the deck has been detected when you connect to the Crazyflie?

**Reply by aelvis237** [reply]
thanks very much. the Problem was the Deck .its has  identify the Deck but it seems as if according to it all the nodes are located on a particular position and the Position(0.00,0,00,0.00) keeps counting up and at a particular point its start back.
![image](https://github.com/bitcraze/discussions/assets/62871491/793701dc-d8a5-4ffd-9af8-8a9531c9e672)

**Reply by gemenerik** [reply]
Can you clarify what is going wrong and at what step of the getting started guide?

**Reply by aelvis237** [reply]
am currently at this level as seen in the picture below
![image](https://github.com/bitcraze/discussions/assets/62871491/0712f36c-d2f6-469b-ae47-db4cfe6866fc)
i have position all the Nodes(8) as you can see in the previous picture .the problem now is that all nodes are identified on a particular position as seen below 
![image](https://github.com/bitcraze/discussions/assets/62871491/654053e2-0b0e-49ec-ac31-972eb239edda)
and when i connect to a crazyflie  this blue point(i think its the crazyflie) 
![image](https://github.com/bitcraze/discussions/assets/62871491/27af8542-0fab-4310-88ca-1b7e5bfd0df7)
keeps moving away from the scene even though am not moving the crazyflie. At first i thought it was trying to determine it current position and i waited for more than 20mins but its keep repeating it( the Position(0.00,0,00,0.00) keeps counting up and at a particular point its start back.)

**Comment by knmcguire** [comment]
Perhaps the pictures on the tutorial is a bit misleading. You should input the positions of the loco anchors at this step of the getting started guide: https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/#enter-anchor-positions. If you just setup your system without setting the positions this is indeed all zero. 

Also if the crazyflies doesn't know the real position of the anchors, it is not able to correctly determine it's position compared to the anchors. The reason you see the position drifting away is because it is not able to converge to a position estimate based on this.

**Reply by HWMathSEU** [reply]
“You should input the positions of the loco anchors at this step”, so you mean I should measure the actual coordinates (for example, the height from the ground) of all the anchors?

**Reply by knmcguire** [reply]
Yes, this is the actual coordinates of the anchors relative to where you'd like the (0, 0, 0) in x,y,z to be. Do mind the coordinate system of the Crazyflie in that sense: https://www.bitcraze.io/documentation/system/platform/cf2-coordinate-system/
